webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py

```diff
@@ -71,10 +70,13 @@ def validate_familienstand_date(self, field):
                 validators.Optional()(self, field)
             elif self.familienstand.data == 'married':
                 validators.InputRequired(_l('validate.date-of-marriage-missing'))(self, field)
+                ValidDateOfMarriage()(self, field)
             elif self.familienstand.data == 'widowed':
                 validators.InputRequired(_l('validate.date-of-death-missing'))(self, field)      
+                ValidDateOfDeath()(self, field)  
             elif self.familienstand.data == 'divorced':
-                validators.InputRequired(_l('validate.date-of-divorce-missing'))(self, field)      
+                validators.InputRequired(_l('validate.date-of-divorce-missing'))(self, field)
+                ValidDateOfDivorce()(self, field) 
```

Comment:
It makes a lot sense to me to split these validators up. However, I think I am not sure why we can use the same validator e.g. for the date of death as well as the date when a couple started living separately before one of them died?

The naming makes a lot of sense for the ones added but not so much for the ones already in place?

Reply:
Thanks julian :+1: Fixed that